documents/MQTT5.md

```diff
@@ -0,0 +1,218 @@
+# MQTT 5
```

Comment:
Debatable, nitpick: I might rename this to MQTT5_Userguide.md just so it's use/contents stand out a bit more than just `MQTT5.md`. Debatable though...